src/includes/getting-started-primer/flutter.mdx

```diff
@@ -4,17 +4,23 @@ _Sentry's Flutter SDK enables automatic reporting of errors, messages, and excep
 
 </Note>
 
+<Note>
+
+Support for Flutter for Desktop/Web is available as a Beta. Features available in Beta are still in-progress and may have bugs. We recognize the irony.
```

Comment:
```suggestion
Support for Flutter's Desktop is available as a Beta. Features available in Beta are still in-progress and may have bugs. We recognize the irony.
```

Reply:
Web support is available since always, so it's not Beta, it's just limited in features